packages/eslint-plugin/tests/rules/consistent-type-imports.test.ts

```diff
@@ -14,6 +14,7 @@ const ruleTester = new RuleTester({
 });
 
 const withMetaParserOptions = {
+  EXPERIMENTAL_useProjectService: false,
```

Comment:
The project service only supports TSConfigs with names like `./tsconfig.json`. So in testing these rules, we explicitly indicate _not_ to use it despite CI potentially setting `process.env.TYPESCRIPT_ESLINT_EXPERIMENTAL_TSSERVER`.

Reply:
Noting - I almost have this working locally I think, just finding issues with the project program not finding the file after I update the root dir...